src/amr/data/particles/refine/split_1d.h

```diff
@@ -1,3 +1,4 @@
+// IWYU pragma: private, include "amr/data/particles/refine/split.h"
```

Comment:
what's that

Reply:
possibly overkill
what this does is, if you want a class that's in split_1d.h, IWYU will tell you that you need "split.h" instead, which I would think is the general case. that you want all dims, not just 1d.

Reply:
ok. I'm just concerned about how consistent with the rest of the code base this is. My understanding is that you ran manually IWYU to get that, so this is something that won't be consistently done. So is it something that we want here?

Reply:
if somethings are considered "private" includes then it might be conceivable to put it in other places.